DataLayerTests/BaseRepositoryTests.cs

```diff
@@ -0,0 +1,143 @@
+using System.Data;
+using System.Data.SqlClient;
+using ADO.NET.DataLayer.Enums;
+using ADO.NET.DataLayer.Models;
+using ADO.NET.DataLayer.Repositories;
+using Moq;
+using Xunit;
+
+namespace DataLayerTests
+{
+    public class BaseRepositoryTests
+    {
+        [Fact]
+        public void ExecuteConnectedQueryWithParams_ShouldOpenConnectionAndExecuteQueryWithParams()
```

Comment:
Actually regular unit tests don't fit here. For such classes as repository integration tests should be used without mocks. You need setup db in docker container. Here https://stackoverflow.com/a/71953236 is good example or you can create docker-compose project and run docker manually like that: https://learn.microsoft.com/en-us/dotnet/architecture/microservices/multi-container-microservice-net-applications/test-aspnet-core-services-web-apps